sqlstore/assets.go

```diff
@@ -112,6 +114,6 @@ func (as *Assets) GetAllWithCursorPagination(ctx context.Context, pagination ent
 }
 
 func getAssetQuery() string {
-	return `SELECT id, name, symbol, total_supply, decimals, quantum, source, erc20_contract, lifetime_limit, withdraw_threshold, vega_time
+	return `SELECT id, name, symbol, total_supply, decimals, quantum, source, erc20_contract, lifetime_limit, withdraw_threshold, vega_time, status
 		FROM assets`
 }
```

Comment:
The GetByID method looks like it has a separate query defined for querying the asset table. We should really reuse the `getAssetQuery()` function but it looks like it has been missed.